### PR TITLE
courses/java: change json format in XML-&-JSON-Data_Formats

### PR DESCRIPTION
**Pull request by tsbonev**
Changed the json format in the last task to include a higher level element that contains the list of people  and added quotation marks so that gson will accept it as proper json.

### CODE REVIEW

File: docs/java/XML-&-JSON-Data-Formats.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ java -jar messagepreview.jar -type json message.json --printAverageStats
 където message.json: 
 
 ```text
```
Comment (mgenov) [inline docs/java/XML-&-JSON-Data-Formats.md]: "```text"  could be replaced with `javascript which should format the json properly.

Comment (mgenov) [reply docs/java/XML-&-JSON-Data-Formats.md]: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code

Comment (tsbonev) [reply docs/java/XML-&-JSON-Data-Formats.md]: Changed text to javascript

**Review by mgenov — APPROVED** [review]
LGTM !

**Pull request merged**